sparcle_qc/sparcle_qc.py

```diff
@@ -375,7 +375,7 @@ def run_sparcle(input_file= None, user_options = None):
 
         keywords = input_parser(input_file)
         #creating new directory for the created files, changing working directories, and copying necessary files into the new directory
-        new_dir = input_file.split('.')[0]
+        new_dir = input_file.split('.in')[0]
```

Comment:
I think we initially did this because we didn't want to mandate the input files end with .in, but this is okay with me too. We should just probably add to the docs that the input file must have the 'in' extension

Reply:
ahh i forgot about that... I'll change it to something that will allow different extensions

Reply:
we could probably just do like input_file.split('.')[:-1]  ?